src/Main.java

```diff
@@ -1,4 +1,94 @@
 public class Main {
     public static void main(String[] args) {
+        System.out.println("Задача № 1");
+        int money = 15000;
+        int total = 0;
+        int month = 0;
+        while (total <= 2_459_000) {
+            month++;
+            total = total + total / 100;
+            total = total + money;
+            System.out.println("Месяц " + month + " сумма накоплений равна " + total + " рублей");
+        }
+
+        {
+            System.out.println("Задача № 2");
+            int a = 1;
+            while (a <= 10) {
+                System.out.print(a + " ");
+                a++;
+            }
+            System.out.println(" ");
+            for (int b = 10; b >= 1; b--) {
+                System.out.print(b + " ");
+            }
+        }
+        {
+            System.out.println("Задача № 3");
+            int totalPeople = 12_000_000;
+            int newPeople = 17 * 12_000;
+            int deadPeople = 8 * 12_000;
```

Comment:
```suggestion
            int newPeople = 17 * totalPeople / 1000;
            int deadPeople = 8 * totalPeople / 1000;
```
используем переменные, а не берём числа с потолка

Reply:
и это надо было считать в цикле, т.к. каждый год популяция меняется и числа разные